common/util/cmd_positional_arguments.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_COMMON_UTIL_CMD_POSITIONAL_ARGUMENTS_H_
+#define VERIBLE_COMMON_UTIL_CMD_POSITIONAL_ARGUMENTS_H_
+
+#include <string>
+#include <vector>
+
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
+
+namespace verible {
+
+class CmdPositionalArguments {
+ public:
+  explicit CmdPositionalArguments(const std::vector<char *> &args)
```

Comment:
We need to state the assumptions about the lifetime of the `char*` here. In the parsing stage, we're putting these in string-views, so we need to remind the user that the `char*` arguments they put in here need to outlive the use-time of this `CmdPositionialArguments` object.

(but also see comment above: the `FileList` struct actually copies verything into a string, so possibly not needed)